### ISSUE TITLE
Julia 1.3.0-rc3 on Windows: "No compression engines found."

### ISSUE THREAD

**Issue by dourouc05** [JuliaLang/julia#33470]
I just installed Julia 1.3.0-rc3 on Windows 10 x64, and I get this error message when installing packages: 

```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.3.0-rc3.0 (2019-10-04)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(v1.3) pkg> add JuMP MathOptInterface CPLEX Xpress Gurobi GLPK Cbc Clp MathOptFormat
  Updating registry at `C:\Users\XXX\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: No compression engines found. We looked for: C:\Users\XXX\AppData\Local\Julia-1.3.0-rc3\bin\7z.exe, 7z, tar, busybox. Install one and ensure it is available on the path.

Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] #probe_platform_engines!#15(::Bool, ::typeof(Pkg.PlatformEngines.probe_platform_engines!)) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\PlatformEngines.jl:488
 [3] probe_platform_engines! at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\PlatformEngines.jl:175 [inlined]
 [4] #download_source#81(::Bool, ::typeof(Pkg.Operations.download_source), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}, ::Dict{Base.UUID,Array{String,1}}) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:611
 [5] #download_source at .\none:0 [inlined]
 [6] #download_source#80 at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:606 [inlined]
 [7] download_source at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:604 [inlined]
 [8] #add#112(::Bool, ::Pkg.BinaryPlatforms.Windows, ::typeof(Pkg.Operations.add), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}, ::Array{Base.UUID,1}) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\Operations.jl:1013
 [9] #add at .\none:0 [inlined]
 [10] #add#25(::Bool, ::Pkg.BinaryPlatforms.Windows, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Pkg.API.add), ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\API.jl:102
 [11] add(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\API.jl:72
 [12] do_add!(::Dict{Symbol,Any}, ::Array{Pkg.Types.PackageSpec,1}, ::Dict{Symbol,Any}) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\REPLMode.jl:505
 [13] #invokelatest#1(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::typeof(Base.invokelatest), ::Any, ::Any, ::Vararg{Any,N} where N) at .\essentials.jl:709
 [14] invokelatest(::Any, ::Any, ::Vararg{Any,N} where N) at .\essentials.jl:708
 [15] do_cmd!(::Pkg.REPLMode.Command, ::REPL.LineEditREPL) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\REPLMode.jl:412
 [16] #do_cmd#23(::Bool, ::typeof(Pkg.REPLMode.do_cmd), ::REPL.LineEditREPL, ::String) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\REPLMode.jl:391
 [17] do_cmd at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\REPLMode.jl:387 [inlined]
 [18] (::Pkg.REPLMode.var"#28#31"{REPL.LineEditREPL,REPL.LineEdit.Prompt})(::REPL.LineEdit.MIState, ::Base.GenericIOBuffer{Array{UInt8,1}}, ::Bool) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\REPLMode.jl:619
 [19] #invokelatest#1 at .\essentials.jl:709 [inlined]
 [20] invokelatest at .\essentials.jl:708 [inlined]
 [21] run_interface(::REPL.Terminals.TextTerminal, ::REPL.LineEdit.ModalInterface, ::REPL.LineEdit.MIState) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\REPL\src\LineEdit.jl:2306
 [22] run_frontend(::REPL.LineEditREPL, ::REPL.REPLBackendRef) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\REPL\src\REPL.jl:1045
 [23] run_repl(::REPL.AbstractREPL, ::Any) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\REPL\src\REPL.jl:201
 [24] (::Base.var"#774#776"{Bool,Bool,Bool,Bool})(::Module) at .\client.jl:390
 [25] #invokelatest#1 at .\essentials.jl:709 [inlined]
 [26] invokelatest at .\essentials.jl:708 [inlined]
 [27] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at .\client.jl:374
 [28] exec_options(::Base.JLOptions) at .\client.jl:312
 [29] _start() at .\client.jl:468

julia> versioninfo()
Julia Version 1.3.0-rc3.0
Commit b5ccd0f2ef (2019-10-04 01:57 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
```

Indeed, the only executable file in Julia's bin folder is Julia itself (even though there is a 7z.dll). I did not get this message when installing the RC2 (but on another computer) or with Julia 1.2, so I guess this is related to packaging. 

A very similar issue: with Julia 1.2, there was `zlib1.dll` packaged along Julia, which is no more the case. Hence GZip.jl stops working. Is this intended or shall I report this to the package? 

I just copied `7z.exe` from my local 7Zip installation, and everything works perfectly.

**Comment by phlavenk** [comment]
Yes, I can confirm the issue on win server 2012R2. Also note that installing 7z into its own folder (in the path) does not help as many packages (GR.jl, ...) call explicitly $(Sys.BINDIR)/7z.exe

**Comment by KristofferC** [comment]
This is a Pkg issue but leaving it here to track with milestone since we should fix it for 1.3.
